### PR TITLE
Initialize project

### PR DESCRIPTION
**Pull request by SaveryIV**
- Added principal components.
- Use of Routes and create of Navbar.
- Create of a simple form.

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
Hi @SaveryIV, please add a description to your pull request description 👍🏿

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <BrowserRouter>
+          <Navbar />
+          <Routes>
+            <Route path="/" element={<Home />} />
+            <Route path="*" element={<NotFound />} />
```
Comment (rica213) [inline src/App.js:19]: - [x] Kudos for handling the not found page 👍🏽

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            <Route path="/categories" element={<BookCollection />} />
```
Comment (rica213) [inline src/App.js]: - [ ] You should update this link with a "Categories" component instead 💯 Kindly note that the **BookCollection** component is not the same as **Categories** component. Your **Categories** component is missing. Please address that 🙏🏽 The **BookCollection** is a component within the Book component 📖
Kindly follow [this visual](https://pedantic-beaver-cb923b.netlify.app/) to have an idea of how the components are organized.

**Review by rica213 — CHANGES_REQUESTED** [review]
# 🟡 STATUS: REQUIRED CHANGES ♻️ 
Hi Agustin
My name is Larissa. Your code reviewer 🥰

Good job so far ❤️ 
However, there are some details that I want you to address 😁 Worry not, you are almost there! 🍄 



Happy coding!

## Highlights

Professional Documentation ✔️
No Linter errors ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._


# Cheers, and Happy coding 🖥️

## If anything is not 100% clear, kindly mention me in your comment and I will reply ASAP 🤞

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (SaveryIV) [reply src/App.js:19]: What is that?

File: src/App.js (hunk 2 above)
Comment (SaveryIV) [reply src/App.js]: My component BookCollection is actually my component categories

Comment (addod19) [reply src/App.js]: - [ ] In my opinion, i think your `BookCollection` component is actually the `BookList` component which renders a list of books using the `<Book>` component, let us kindly create the `Categories` component as mentioned in the previous code review

**Review by addod19 — CHANGES_REQUESTED** [review]
## Hi @SaveryIV, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- [ ] Nice job with this project :+1: 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @SaveryIV,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ Linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**